planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -1859,6 +1859,8 @@ PathWithLaneId getCenterLinePath(
     s_forward = std::min(s_forward, goal_arc_coordinates.length - lane_change_buffer);
   }
 
+  s_forward = std::max(s_forward, s_backward);
```

Comment:
This makes sense.
I would suggest use if else and put `RCL_WARN` to inform the user that the goal placement for lane change is invalid.